charts/fleet-telemetry/templates/2-deployment.yaml

```diff
@@ -27,6 +27,10 @@ spec:
         env:
         - name: KUBERNETES_CLUSTER_DOMAIN
           value: {{ quote .Values.kubernetesClusterDomain }}
+        {{- if .Values.gcpSecret }}
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/gcp/key.json
```

Comment:
shouldn't this be `/etc/gcp/{{.Values.gcpSecret}}`
I dont know the proper syntax for it